attest/verifier/src/status.rs

```diff
@@ -55,43 +74,17 @@ pub enum Kind {
 }
 
 impl Kind {
-    /// Assume an enclave with the specified measurement does not need
-    /// BIOS configuration changes to address the provided advisory IDs.
-    ///
-    /// This method should only be used when advised by an enclave author.
-    pub fn allow_config_advisories<'a>(
-        &mut self,
-        ids: impl IntoIterator<Item = &'a str>,
-    ) -> &mut Self {
-        for id in ids {
-            match self {
-                Kind::Enclave(v) => {
-                    v.allow_config_advisory(id);
-                }
-                Kind::Signer(v) => {
-                    v.allow_config_advisory(id);
-                }
-            }
-        }
-        self
-    }
-
     /// Assume an enclave with the specified measurement has the appropriate
     /// software/build-time hardening for the given advisory IDs.
```

Comment:
:wrench: Seems like this comment should be updated, the "software/build-time hardening" seems to imply `SwHardening`
```suggestion
    /// Advisories an enclave is allowed to have.
```